frontend/babel.config.js

```diff
@@ -2,6 +2,7 @@ module.exports = function (api) {
   api.cache(true);
   return {
     presets: ["babel-preset-expo"],
-    plugins: ["module:react-native-dotenv"],
+    plugins: ["module:react-native-dotenv", "react-native-paper/babel"],
+    plugins: ["react-native-paper/babel"],
```

Comment:
also we can prob delete this second plugins line? seems redundant but idk if I'm missing something